keyboards/dichotomy/keymaps/alairock/keymap.c

```diff
@@ -0,0 +1,205 @@
+// this is the style you want to emulate.
+// This is the canonical layout file for the Quantum project. If you want to add another keyboard,
+
+#include "dichotomy.h"
```

Comment:
This should actually be:

```suggestion
#include QMK_KEYBOARD_H
```

Not your fault really; the default keymap needs to be updated as well.